segment-kube.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+type KubeContext struct {
+	Context struct {
+		Cluster   string
+		Namespace string
+		User      string
+	}
+	Name string
+}
+
+type KubeConfig struct {
+	Contexts       []KubeContext `yaml:"contexts"`
+	CurrentContext string        `yaml:"current-context"`
+}
+
+func segmentKube(p *powerline) {
+	home, _ := os.LookupEnv("HOME")
+	defaultConfigFile := home + "/.kube/config"
```

Comment:
use `path.Join()` for windows support (this codebase does have windows support, it appears)

Reply:
Yes, surprisingly, it does have Windows support – and apparently, it actually works on Windows.

Reply:
I agree, this should be changed.